### PR TITLE
Video script branch

### PR DESCRIPTION
**Pull request by Eleder143**
added a new checkbox that, when checked, makes a new textarea appear in which you can write the script of the video. It still needs subject to find words for the downloaded background videos.

### CODE REVIEW

**Comment by FujiwaraChoki** [comment]
Nice, make sure to testing and also, add remove the `print('aaaaaaaayow')`

File: Backend/gpt.py [hunk 1]
```diff
@@ -60,6 +62,7 @@ def generate_script(video_subject: str) -> str:
 
 
 def get_search_terms(video_subject: str, amount: int, script: str) -> List[str]:
+    print('aaaaaaaayow')
```
Comment (Arne1303) [inline Backend/gpt.py:65]: I think this line could be removed

File: Backend/gpt.py [hunk 2]
```diff
@@ -5,7 +5,9 @@
 from termcolor import colored
 
 
-def generate_script(video_subject: str) -> str:
+def generate_script(video_subject: str, video_script: str) -> str:
+    if video_script:
+        return video_script
```
Comment (Arne1303) [inline Backend/gpt.py:10]: Why not check at the point where we call the function? This seems a bit out of place

**Comment by Arne1303** [comment]
Not a reviewer, still wanted to say some stuff at the end @FujiwaraChoki has to decide

File: Backend/gpt.py (hunk 1 above)
Comment (FujiwaraChoki) [reply Backend/gpt.py:65]: Agree

**Comment by FujiwaraChoki** [comment]
Please resolve conflicts and add/remove proposed changes. Other than that it looks fine. Thank you!

**Comment by Eleder143** [comment]
All done masters!

**Pull request closed without merging**